### ISSUE TITLE
multiple() not working

### ISSUE THREAD

**Issue by Jaspur** [optimistdigital/nova-media-field#31]
Hi @marttinnotta & @Tarpsvo,

The `multiple()` config option doesn't work anymore. I only can select one.

```  
MediaField::make('Images')
                ->collection('foobar')
                ->multiple();
```

Is this a bug or a feature? ;-) 

Tried v1 and v2 codebase. Both same issue.

**Comment by Jaspur** [comment]
Tried the package on a different project; same bug.

Both on last versions of Nova & Laravel.
Also tried it with PHP8 & 7.4

@marttinnotta @Tarpsvo

**Comment by Wndworks** [comment]
@Jaspur @marttinnotta @Tarpsvo I had the same issue two weeks ago but couldn't solve it. Any help would be nice.

**Comment by marttinnotta** [comment]
This should now be fixed in 1.4.1 release.

**Comment by Jaspur** [comment]
@marttinnotta Thanks! Works after resetting the cache.
